app/jobs/github_commits_job.rb

```diff
@@ -8,6 +8,7 @@ def run
       begin
         update_total_commit_count_for(project)
         update_user_commit_counts_for(project)
+        update_last_pushed_dt_for(project)
       rescue Exception
```

Comment:
And we might want to replace this Exception with StandardError: https://robots.thoughtbot.com/rescue-standarderror-not-exception I know you didn't introduce this, but a great time to fix it perhaps ...